### PR TITLE
sr-iov operator bases for 4.10

### PR DESCRIPTION
**Pull request by jnunyez**

### CODE REVIEW

**Review by natifridman — COMMENTED** [review]
Looks good, I would also add a readme with an example of how to use it. Something like [this](https://github.com/redhat-cop/gitops-catalog/tree/main/advanced-cluster-management/operator)

**Comment by jnunyez** [comment]
> Looks good, I would also add a readme with an example of how to use it. Something like [this](https://github.com/redhat-cop/gitops-catalog/tree/main/advanced-cluster-management/operator)

Added howto use it

File: components/apps/sriov-operator/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# OpenShift Single Root I/O Virtualization Operator
+
+Installs the SR-IOV Operator.
+
+Do not use the `base` directory directly, as you will need to patch the `channel` based on the version of OpenShift you are using, or the version of the operator you want to use.
+
+The current *overlays* available are for the following channels:
+* [4.10](overlays/4.10)
```
Comment (natifridman) [inline components/apps/sriov-operator/README.md:8]: 4.11 also
```suggestion
* [4.10](overlays/4.10)
* [4.11](overlays/4.11)
```

**Review by natifridman — CHANGES_REQUESTED** [review]

**Review by natifridman — APPROVED** [review]
LGTM, looks like you need to rebase with main branch

**Pull request merged**